src/design-system/components/alert/Alert.tsx

```diff
@@ -0,0 +1,96 @@
+import { SyntheticEvent, forwardRef, ButtonHTMLAttributes } from 'react'
+import {
+  Alert as MuiAlert,
+  AlertProps as MuiAlertProps,
+  AlertTitle as MuiAlertTitle,
+  AlertTitleProps
+} from '@mui/material'
+import {
+  ErrorOutline,
+  WarningAmberOutlined,
+  InfoOutlined,
+  CheckCircleOutline,
+  CloseRounded
+} from '@mui/icons-material'
+
+import { cn } from '~/utils/cn'
+
+import '~scss-components/alert/Alert.scss'
+
+export const AlertTitle = ({ children, ...props }: AlertTitleProps) => {
+  return <MuiAlertTitle {...props}>{children}</MuiAlertTitle>
+}
+
+AlertTitle.displayName = 'AlertTitle'
+
+interface CloseButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
+  label?: string
+}
+
+const CloseButton = ({ label }: CloseButtonProps) => {
+  return (
+    <button aria-label='Close alert' className='s2s-alert-close-button'>
+      {label && <span className='s2s-alert-close-button-label'>{label}</span>}
+      <CloseRounded />
+    </button>
+  )
+}
+
+CloseButton.displayName = 'CloseButton'
+
+const iconMapping = {
+  error: <ErrorOutline fontSize='inherit' />,
+  warning: <WarningAmberOutlined fontSize='inherit' />,
+  info: <InfoOutlined fontSize='inherit' />,
+  success: <CheckCircleOutline fontSize='inherit' />
+}
+
+interface AlertProps extends MuiAlertProps {
+  title?: string
+  description?: string
+  label?: string
+}
+
+const Alert = forwardRef<HTMLDivElement, AlertProps>(
+  (
+    {
+      title,
+      label,
+      description,
+      children,
+      icon,
+      onClose,
+      className,
+      ...props
+    }: AlertProps,
```

Comment:
```suggestion
    },
```
It is already typed by generic forwardRef